doc/cpp/interconnectivity.rst

```diff
@@ -45,13 +45,13 @@ Interconnectivity
     a cell or a spike source. The target is a synapse on the post-synaptic cell.
 
     The :cpp:member:`target` does not include the gid of a cell, this is because a
-    :cpp:class:`ext_cell_connection` is bound to the target cell which means that the gid
+    :cpp:class:`ext_cell_connection` is bound to the target cell, which means that the gid
     is implicitly known.
 
     .. cpp:member:: cell_remote_label_type source
 
         Source end point, represented by a :cpp:type:`cell_remote_label_type` which packages
```

Comment:
Agreed, please advise to make it better. How about:

Source endpoint, comprising
- global id; uniquely identifying the source object inside the remote simulation
- local id; uniquely identifying the location of the source on the source object
- selection policy; how to map sources `(gid, lid)` to the targets in Arbor.

Note: The tuple `(gid, lid)` must be unique for each source and follows Arbor's model of multiple
sources (threshold detectors) per cell. If the remote simulation does not provide multiple items per
source object, `lid` can be ignored and set to zero.